response_operations_ui/templates/close-conversation.html

```diff
@@ -26,7 +26,7 @@ <h1 class="u-fs-xl">Close conversation</h1>
           Reference:
         </dt>
         <dd class="">
-          {{ru_ref}}
+          {{display_ru_ref}}
```

Comment:
I think it'd be fine to leave this as is and just add the ru_ref_filter to distinguish between the two?  Just an opinion

Reply:
Your right, this was changed to this as at one point I was not using ru_ref_filter but ru_ref, But since I changed to use _fliter variables , this can also change back for clarity